src/black/__init__.py

```diff
@@ -6632,7 +6605,7 @@ def can_omit_invisible_parens(
             # unnecessary.
             return True
 
-        if line.should_explode and penultimate.type == token.COMMA:
+        if line.should_split and penultimate.type == token.COMMA:
```

Comment:
Just highlighting that there is something slightly unusual related to this condition.  The left-hand-side of this condition is removed in #1963.

Reply:
(this conditional block has now been removed)